opass

```diff
@@ -257,8 +257,11 @@ def main():
     # when doing so is part of its job.
     online_p = True  # default assumption
 
+    verbose = False
+
     try:
-        (opts, args) = getopt.getopt(sys.argv[1:], "ho", ["help", "offline",])
+        (opts, args) = getopt.getopt(sys.argv[1:], "hov",
+                ["help", "offline", "verbose"])
```

Comment:
Don't forget comma after last item in tuple -- not strictly necessary, but good form and protects against oddball mis-parsings that can happen in edge cases.

Reply:
This is a list, not a tuple, so I don't think the same mis-parsings that can happen with a single-item tuple are likely here, but nevertheless I re-added the comma for consistency with the rest of the file.

Reply:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83